nncf/experimental/openvino_native/statistics/collectors.py

```diff
@@ -0,0 +1,113 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import Union, List, Deque
+
+import numpy as np
+
+from nncf.common.tensor import NNCFTensor
+from nncf.common.tensor import TensorElementsType
+from nncf.common.tensor_statistics.collectors import MinMaxStatisticCollector
+from nncf.common.tensor_statistics.collectors import NNCFCollectorTensorProcessor
+from nncf.common.tensor_statistics.collectors import MeanMinMaxStatisticCollector
+from nncf.common.tensor_statistics.collectors import MeanStatisticCollector
+from nncf.experimental.openvino_native.tensor import OVNNCFTensor
+from nncf.experimental.openvino_native.statistics.statistics import OVMinMaxTensorStatistic
+from nncf.experimental.openvino_native.statistics.statistics import OVMeanTensorStatistic
+
+
+class OVNNCFCollectorTensorProcessor(NNCFCollectorTensorProcessor):
```

Comment:
Did I get right that we use an external statistic collection mechanism now? Any plans for the in-place statistics?

Reply:
@l-bat, could you file a ticket based on this comment?

Reply:
ticket: 100927